server/migrations/20211102143133-add manualJob_meta column on job_execution table.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
Why is this required? I thought we decided to use the new JSON field created in the Job table by Kostas to store any metadata